.gitattributes

```diff
@@ -0,0 +1,8 @@
+###############################
```

Comment:
Missing SPDX identifier in both files.

Reply:
I added the SPDX line in both. Shall we do this for every file? For example, gitignore also doesn't have this.